README.md

```diff
@@ -26,25 +26,12 @@ Metrics are collected from the following components:
 * API server
 * Prometheus (just `prometheus-k8s` for now)
 * Alertmanager
-* Telemeter
 
-## Contributing new component integrations
+## Adding new metrics to be sent via telemetry
 
-The Cluster Monitoring Operator has many builtin `ServiceMonitor` resources which enable discovering the metrics endpoints of a variety of well-known components. Only components that must be created before the cluster monitoring stack belong in this repository, in order to solve the cyclic dependencies of bootstrapping.
+To add new metrics to be sent via telemetry, simply add a selector that matches the time-series to be sent in [manifests/0000_50_cluster_monitoring_operator_04-config.yaml](manifests/0000_50_cluster_monitoring_operator_04-config.yaml).
```

Comment:
This doesn't need any regeneration of the files?

Reply:
Nope, this will be the only file people will need to modify to add a new metric.